bigchaindb/commands/bigchaindb.py

```diff
@@ -163,13 +173,17 @@ def run_upsert_validator_approve(args, bigchain):
     if len(voting_powers) > 0:
         voting_power = voting_powers[0]
     else:
-        raise KeypairMismatchException(
-            'The key you provided does not match any of the eligible voters in this election.'
-        )
+        logger.error('The key you provided does not match any of the eligible voters in this election.')
+        return False
 
+    # NOTE: The code below assumes no vote delegation has taken place. Vote delegation
```

Comment:
BEP-21 still mentions delegation:

> A validator can choose to vote/delegate/burn their vote.

The mention is not a bold one but might be better to remove it.

Reply:
Is there a reason we should remove it? The `approve` command doesn't support delegation but that was never the scope for `approve` command.

Reply:
> we should not mention this inside the approve code .
Any particular reason you recommend doing that?

Reply:
If delegation is not supported by the current implementation, I suggest to remove the `A validator can choose to vote/delegate/burn their vote.` note from BEP-21 because it suggests the opposite.

If delegation is supported but the `approve` command is simply not concerned with it, I suggest to remove the `#NOTE` comment because it suggests that the `approve` function actually needs to be adopted to make delegation work.